frontend/src/app/firewood/forests/order-confirmation/order-confirmation.component.html

```diff
@@ -0,0 +1,94 @@
+<section id="application-view" class="usa-section usa-grid">
+  <app-breadcrumbs></app-breadcrumbs>
+  <app-tree-progress-bar step="print"></app-tree-progress-bar>
+  <div>
+    <div>
+      <h1 id="skip-nav-target" tabindex="-1" class="usa-heading">Firewood permit confirmation</h1>
+    </div>
+
+    <div class="confirmation-details-container margin-top-2">
+      <p>Your firewood permit has been purchased.</p>
+      <p>Your permit, load tags and required documents have been sent to <strong>email@address.com</strong>. If for some reason this email address is incorrect, please print your documents now.</p>
+      <p><strong>You will need to have your permit, load tags and required documents with you when collecting firewood for your permit to be valid.</strong> Your load tags will have instructions for how to fill it out and attach it to your load.</p>
+    </div>
+
+    <div class="required-documents">
+      <h2>Required documents</h2>
+      <ul>
+        <li>Your Permit</li>
+        <li>Your load tags</li>
+        <li>Cutting map</li>
+      </ul>
+      <input id="includeRules" type="checkbox" name="includeRules" [checked]="includeRules" (change)="includeRules = !includeRules">
+      <label for="includeRules">Include a copy of the firewood cutting regulations</label>
+      <button id="tree-print-permit" class="usa-button-primary usa-button-big" type="button" [disabled]="isPermitExpired" (click)="printPermit()">
+        Print now
+      </button>
+      <p>These documents have also been emailed to <strong>email@address.com</strong></p>
+    </div>
+
+    <div *ngIf="forest" id='tree-order-summary-container' class="confirmation-details-container clear margin-top-2">
+      <div *ngIf="isPermitExpired" class="usa-alert usa-alert-error permit-expiration-warning">
+        <div class="usa-alert-body">
+          <!-- <h3 class="usa-alert-heading">This permit was issued on {{permit.transactionDate | date: 'shortDate'}} and expired on {{permit.expirationDate | date: 'shortDate'}}</h3> -->
```

Comment:
It looks like there are several commented lines on this page that we might not need. Can we remove them or do they need to be there as a just-in-caser?

The other comments are on lines: 48, 60, 70, 79, 88, and 90

Reply:
These lines are temporarily commented out until we have the `GET` request for the permit information working. Right now they won't populate correctly, but we will need them.